js/core/config.d.ts

```diff
@@ -15,14 +15,14 @@ declare function config(): globalConfig;
 
 /**
  * @docid
- * @publicName config(config)
+ * @publicName config(configuration)
```

Comment:
This change affects topics generation

```suggestion
 * @publicName config(config)
```